README-dev.md

```diff
@@ -1,80 +1,134 @@
-# How to contribute to the o1js codebase
+# o1js README-dev
 
-This README includes information that is helpful for o1js core contributors.
+o1js is a TypeScript framework designed for zk-SNARKs and zkApps on the Mina blockchain.
 
-## Run examples using Node.js
+- [zkApps Overview](https://docs.minaprotocol.com/zkapps)
+- [Mina README](/src/mina/README.md)
 
-```sh
-npm install
-npm run build
+For more information on our development process and how to contribute, see [CONTRIBUTING.md](https://github.com/o1-labs/o1js/blob/main/CONTRIBUTING.md). This document is meant to guide you through building o1js from source and understanding the development workflow.
+
+## Prerequisites
+
+Before starting, ensure you have the following tools installed:
 
-./run src/examples/api_exploration.ts
+- [Git](https://git-scm.com/)
+- [Node.js and npm](https://nodejs.org/)
+- [Dune](https://github.com/ocaml/dune)
+- [Cargo](https://www.rust-lang.org/learn/get-started)
```

Comment:
I don't want to scare JS devs away from contributing -- maybe you could add a qualification here, similar to this:
```suggestion
- [Dune](https://github.com/ocaml/dune) (only needed when compiling o1js from source)
- [Cargo](https://www.rust-lang.org/learn/get-started) (only needed when compiling o1js from source)
```